### PR TITLE
Adding new nextra icon in markdown badge

### PR DESCRIPTION
**Pull request by officialrajdeepsingh**
## Description

Nextra is a static site generator based on Next.js.
Adding new [nextra](https://nextra.site/) icon in markdown badges and checkout the [nextra GitHub repository](https://github.com/shuding/nextra).

## Info

|Name|Category|Background Color|Logo Color|
|:--:|:--:|:--:|:--:|
Nextra| 📚 Frameworks, Platforms and Libraries| `#000000`|`#ffffff`

## Preview

| Badge | url |
| ----- | --- |
| Nextra | `Nextra` |

### CODE REVIEW

**Review by kohasummons — COMMENTED** [review]
@officialrajdeepsingh Thanks for contributing ✨ , rajdeep. I just reviewed your PR and noticed the badge is missing the Nextra Icon. Can you please check it out 🙏🏽

File: README.md [hunk 1]
```diff
@@ -427,6 +427,7 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
...
 | NestJS             | ![NestJS](https://img.shields.io/badge/nestjs-%23E0234E.svg?style=for-the-badge&logo=nestjs&logoColor=white)                                    | `![NestJS](https://img.shields.io/badge/nestjs-%23E0234E.svg?style=for-the-badge&logo=nestjs&logoColor=white)`                                    |
 | Next JS            | ![Next JS](https://img.shields.io/badge/Next-black?style=for-the-badge&logo=next.js&logoColor=white)                                            | `![Next JS](https://img.shields.io/badge/Next-black?style=for-the-badge&logo=next.js&logoColor=white)`                                            |
+| Nextra             | ![Nextra](https://img.shields.io/badge/nextra-black.svg?style=for-the-badge&logo=nextra&logoColor=white)                                        | `![Nextra](https://img.shields.io/badge/nextra-black.svg?style=for-the-badge&logo=nextra&logoColor=white)`                                        |
```
Comment (officialrajdeepsingh) [inline README.md:430]: Icons are already there and tell what is missing

Comment (officialrajdeepsingh) [reply README.md:430]: What is the meaning of badge

Comment (kohasummons) [reply README.md:430]: image

A snapshot from the Nextra webpage. Sadly, the Nextra Icon is missing from the badge. 
A badge looks like thisNext JS

Comment (officialrajdeepsingh) [reply README.md:430]: Somehow, the `https://img.shields.io/badge` API does not return the nextra SVG icon and API like `https://unpkg.com/simple-icons@v9/icons/nextra.svg` is work fine.

**Pull request closed without merging**